microdnf/commands/download/download.cpp

```diff
@@ -86,7 +86,12 @@ void CmdDownload::run(Context & ctx) {
         result_pset |= solv_query.get_package_set();
     }
 
-    download_packages(result_pset, ".");
+    std::vector<libdnf::rpm::Package> download_pkgs;
+    download_pkgs.reserve(result_pset.size());
+    for (auto package : download_pkgs) {
```

Comment:
I suggest that you want to iterate thought `result_pset`

Reply:
Yes, you are right. Thank you. I pushed the bad version.